testsuite/MDAnalysisTests/analysis/test_contacts.py

```diff
@@ -171,16 +171,64 @@ class TestContacts(object):
     def universe():
         return mda.Universe(PSF, DCD)
 
-    def _run_Contacts(self, universe, start=None, stop=None, step=None, **kwargs):
+    def _run_Contacts(self, universe, sel_acidic=None, sel_basic=None, start=None, stop=None, step=None, **kwargs):
```

Comment:
Can i just query why the choice of `sel_acidic` and `sel_basic`? perhaps I'm missing some context here.

Reply:
Thanks for all your comments! Let me answer this while I work on implementing pystest.raises(). 
I initially thought of using `ag_acidic` and `ag_basic`, where `ag` is for AtomGroup, but then I realized it doesn't have to be an AtomGroup, and can be a selection text. I thought `sel` would be more general. However, now I realize that it can be easily confused with `self.sel_basic` and `self.sel_acidic`. Should I change the variable names to `ag_*` instead of `sel_*`?

Reply:
yeah that makes sense.